tests/test_bukuDb.py

```diff
@@ -40,7 +40,8 @@
      "a case for replace_tag test"],
 ]
 
-only_python_3_5 = pytest.mark.skipif(sys.version_info < (3, 5), reason="requires python3.5")
+only_python_3_5 = pytest.mark.skipif(
+    sys.version_info < (3, 5), reason="requires minimal python3.5")
```

Comment:
"requires minimal python3.5"

should be 

    "requires Python 3.5 or later"